packages/web-components/src/components/ic-navigation-menu/ic-navigation-menu.css

```
@@ -82,9 +82,9 @@
 
 .navigation-landmark-text {
   position: absolute;
-  width: 0.063rem;
-  height: 0.063rem;
+  width: var(--ic-space-1px);
+  height: var(--ic-space-1px);
   padding: 0;
-  margin: -0.063rem;
+  margin: -var(--ic-space-1px);
```

This will need to be converted like you did with some of the others
```suggestion
  margin: calc(-1 * var(--ic-space-1px));
```